python/ray/_raylet.pyx

```diff
@@ -244,6 +253,18 @@ cdef unordered_map[c_string, double] resource_map_from_dict(resource_map):
     return out
 
 
+cdef c_vector[c_string] string_vector_from_list(string_list):
+    cdef:
+        c_vector[c_string] out
+    if not isinstance(string_list, list):
```

Comment:
you can actually do `cdef c_vector[c_string] string_vector_from_list(list string_list)` here